src/internal/task/task_stack_cortexm.go

```diff
@@ -0,0 +1,83 @@
+// +build scheduler.tasks, cortexm
```

Comment:
I discovered an issue here while playing around with `-scheduler=none`: this file is still included. Removing the extra space (to make it `scheduler.tasks, cortexm`) fixes the issue.